subprojects/environments/spark/spark-env.yml

```diff
@@ -0,0 +1,37 @@
+name: ipython-spark
+cluster:
+  docker:
```

Comment:
Perhaps it would make sense to name this node "master" or "notebook", since "docker" doesn't necessarily have a meaning to the user (i.e users may wonder what the "docker" node is for)